charts/tidb-cluster/values.yaml

```diff
@@ -396,6 +397,7 @@ tidb:
   service:
     type: NodePort
     exposeStatus: true
+    portName: "mysql-client"
```

Comment:
```suggestion
    portName: "mysql"
```

Reply:
For compatibility, should we use the original portName？